spec/vcr_setup.rb

```diff
@@ -0,0 +1,41 @@
+require 'vcr'
+require 'rspec'
+
+module BlockScore
+  VCR.configure do |c|
+    c.cassette_library_dir = 'spec/cassettes'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.